faq.md

```diff
@@ -0,0 +1,113 @@
+
+## What should I bring to the hackathon?
+Participants should bring their own laptops, chargers, and any other tools or software they require for coding. The hackathon organizers will provide the space, power, and Internet required for the competition.
+
+## What programming languages and technologies are allowed?
+The hackathon is technology-agnostic, meaning you are free to choose any tool, modern programming language used by AI tools, framework, or technology stack to develop your AI solution. The focus is on creativity, innovation, and the practical maintainable implementation of AI concepts.
+
+## Are there any tools that are offering credits?
+Coming Soon
+
+## How long is the hackathon?
+The TED AI Hackathon will span 30 hours starting at 10am on Oct 19, 2023 and ending at 4pm on Oct 20, 2023.
+
+## Will there be technical support available?
+We will provide space, power, and Internet connectivity for the hackathon. There will be some limited mentoring and technical support available. Your team should have all the skills to troubleshoot any technical issues otherwise.
+
+## What are the judging criteria for the hackathon?
+The judging criteria for the hackathon will consider factors such as innovation, technical complexity, practicality, user experience, and the overall quality of the AI solution developed during the hackathon. We will share the rubric that will be used by judges to evaluate participation. This will be shared closer to the hackathon date.
+
+## Will cloud credits be provided to TED AI Hackathon participants?
+More details about cloud access will be provided closer to the hackathon date.
+
+## Are there any prizes for the winners?
+Coming Soon
+
+## Can I attend the hackathon even if I am not participating?
+The hackathon is only intended for registered participants.
+
+## How can I stay updated on hackathon announcements?
+To stay updated on hackathon announcements, make sure to regularly check the official TED AI conference website, follow the TED AI social media accounts, and subscribe to the TED AI newsletter.  
+
+If you have any further questions or need additional information, please do not hesitate to reach out to the hackathon core team using tedaihackathon@xy.ai.
+
+## What data may I use?
+Data should come from public sources with Creative Commons or permissive licenses or cited according to the data creator’s/publisher’s guidance. The data used in the hackathon must be accessible to a global audience. 
+
+## Where will the code reside and what licenses will be considered?
+LF AI & Data Foundation has created a GitHub organization. We recommend all participants publish their source code as a repo to this organization. The code must use an OSI-approved license.
+
+## How do I contact organizers about the hackathon?
+Send us an email at tedaihackathon@xy.ai.
```

Comment:
Change this to TEDAI.hackathon@gmail.com

Reply:
Addressed in 75a5b311cac6e25ccad6dade58ace598e559553d